### PR TITLE
Build docs

### PR DESCRIPTION
**Pull request by lifekaizen**
Seem to need to have the action in main to test. Not seeing this: https://docs.github.com/en/actions/quickstart#viewing-your-workflow-results

### CODE REVIEW

File: packages/collabswarm/package.json [hunk 1]
```diff
@@ -23,9 +23,9 @@
     "jest": "^27.0.4",
     "ts-jest": "^27.0.3",
     "ts-node": "^10.1.0",
-    "typedoc": "^0.21.0",
+    "typedoc": "<0.21.0",
     "typedoc-neo-theme": "^1.1.1",
-    "typescript": "^4.3.5"
+    "typescript": "<4.3"
```
Comment (robotoer) [inline packages/collabswarm/package.json:28]: What's the reason for this (and the change above it)?

**Review by robotoer — APPROVED** [review]
A few other comments:

- Currently we only build documentation for the `@collabswarm/collabswarm` package. Would be nice to generate docs for the other sites
- Would be nice to trigger the `typedoc` build from github actions as well. Each action can run arbitrary shell commands too

**Comment by lifekaizen** [comment]
> * Currently we only build documentation for the `@collabswarm/collabswarm` package. Would be nice to generate docs for the other sites

Should be able to add that. Let me get this working first, and think about what the publish URL would be.

> * Would be nice to trigger the `typedoc` build from github actions as well. Each action can run arbitrary shell commands too

[This file](https://github.com/collabswarm/collabswarm/blob/build-docs/.github/worflows/typedoc.yml) should accomplish that, difficult to test it

File: packages/collabswarm/package.json (hunk 1 above)
Comment (lifekaizen) [reply packages/collabswarm/package.json:28]: Fix build error, see issue in commit msg.

**Pull request merged**